pkg/controller/certificatesigningrequests/selfsigned/selfsigned.go

```diff
@@ -43,6 +43,7 @@ import (
 	"github.com/jetstack/cert-manager/pkg/util/pki"
 )
 
+// CSRControllerName holds the controller name
```

Comment:
suggestion: I think this comment should be _inside_ the const block, directly above `CSRControllerName`

Reply:
done